scala-conductr-client-lib/src/main/scala/com/typesafe/conductr/clientlib/scala/models/BundleAttributes.scala

```diff
@@ -13,6 +13,7 @@ final case class BundleAttributes(
   roles: SortedSet[String],
   bundleName: String,
   systemVersion: String,
-  compatibilityVersion: String
+  compatibilityVersion: String,
+  tags: Option[Seq[String]]
```

Comment:
@huntc - if we want CD to be backward compatible between 2.1 and 2.0, I think the client lib has to be backward compatible with 2.0 as well.

Otherwise if we were to create 2 different versions of client lib, it would also mean 2 different versions of CD bundles for 2.1 and 2.0 respectively.

I think making the clientlib (and hence the CD bundle) backward compatible, is the easier path going forward.

Wdyt?